### PR TITLE
Deploying a DMS in the cloud (DaaS)

### PR DESCRIPTION
**Pull request by MariekeGO**

### CODE REVIEW

File: user-guide/Getting_started/Deploying_a_DMS_in_the_cloud.md
Comment (BertVandenberghe) [inline user-guide/Getting_started/Deploying_a_DMS_in_the_cloud.md:1]: Can you replace daas@skyline.be with daas@dataminer.services? We'll those the dataminer.services domain. in this context. Thanks!

Comment (MariekeGO) [reply user-guide/Getting_started/Deploying_a_DMS_in_the_cloud.md:1]: OK, done!

File: user-guide/Getting_started/Deploying_a_DataMiner_System.md
Comment (BertVandenberghe) [inline user-guide/Getting_started/Deploying_a_DataMiner_System.md:1]: We don't use a database cluster, so maybe rephrase the STaaS part to make clear we use cloud native storage.

File: user-guide/Getting_started/Installing_a_DMA/Installing_DM_using_the_DM_Installer.md
Comment (BertVandenberghe) [inline user-guide/Getting_started/Installing_a_DMA/Installing_DM_using_the_DM_Installer.md:1]: We don't use storage nodes for STaaS but cloud native storage, maybe rephrase this to reflect this.

File: user-guide/Getting_started/Installing_a_DataMiner_Agent.md
Comment (BertVandenberghe) [inline user-guide/Getting_started/Installing_a_DataMiner_Agent.md:1]: We don't use storage nodes for STaaS but cloud native storage, maybe rephrase this to reflect this.

**Review by BertVandenberghe — COMMENTED** [review]
added a couple of improvements, can you take a look at those? thx

**Comment by MariekeGO** [comment]
> added a couple of improvements, can you take a look at those? thx

The necessary changes have been implemented. Could you take a look?

**Review by BertVandenberghe — APPROVED** [review]
Perfect, thanks!
